### PR TITLE
Workspaces feature

### PR DESCRIPTION
**Pull request by acamacho82**

### CODE REVIEW

File: features/workspace.feature [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    When I requested delete it
+    Then verify the deletion status is 404
+    And the response reason is "Not Found"
+
+  Scenario: Delete all workspaces
+    Given I delete all the workspaces
+    Then verify the list is empty
+
+
+# Feature: Work with projects into workspaces
```
Comment (sperBruno) [inline features/workspace.feature]: Delete comments

File: tests/pivotal_api_services/workspace_tests.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    @staticmethod
+    def _workspace_creation_failed():
+        response = {
+            "code": "invalid_parameter",
+            "error": "One or more request parameters was missing or invalid."
+        }
+        return response
+
+
+if __name__ == '__main__':
```
Comment (sperBruno) [inline tests/pivotal_api_services/workspace_tests.py]: this is not required, delete it

**Review by sperBruno — CHANGES_REQUESTED** [review]

File: features/workspace.feature (hunk 1 above)
Comment (acamacho82) [reply features/workspace.feature]: delete them

File: tests/pivotal_api_services/workspace_tests.py (hunk 2 above)
Comment (acamacho82) [reply tests/pivotal_api_services/workspace_tests.py]: delete them and I added more unittests

**Comment by acamacho82** [comment]
hmm this should not merged as it is pointing to `master` instead of `develop`

**Pull request closed without merging**